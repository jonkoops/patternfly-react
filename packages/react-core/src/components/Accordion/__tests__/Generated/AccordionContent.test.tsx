/**
 * @vitest-environment jsdom
 */
import { render } from '@testing-library/react';
import React from 'react';
import { expect, it } from 'vitest';

import { AccordionContent } from '../../AccordionContent';

it('AccordionContent should match snapshot (auto-generated)', () => {
  const { asFragment } = render(
    <AccordionContent
      children={<>ReactNode</>}
      className={"''"}
      id={"''"}
      isHidden={false}
      isFixed={false}
      aria-label={"''"}
      component={() => <div />}
    />
  );
  expect(asFragment()).toMatchSnapshot();
});
