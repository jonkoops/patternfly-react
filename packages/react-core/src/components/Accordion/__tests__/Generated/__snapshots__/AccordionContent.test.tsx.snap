// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`AccordionContent should match snapshot (auto-generated) 1`] = `
<DocumentFragment>
  <div />
</DocumentFragment>
`;
